packages/@sanity/dashboard/src/widgets/sanity-tutorials/index.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react'
+import sanityClient from '@sanity/client'
+import imageUrlBuilder from '@sanity/image-url'
+import styles from './index.css'
+import Tutorial from './Tutorial'
+import {get} from 'lodash'
+import {distanceInWords} from 'date-fns'
+const client = sanityClient({
```

Comment:
Do we really want to expose this directly as a GROQ-query? If so, we should probably _at least_ add a comment explaining what this project ID is all about.

Reply:
I agree. This should be config passed to the widget, yeah?